deploy/examples/create-external-cluster-resources.py

```diff
@@ -1248,16 +1248,7 @@ def validate_rados_namespace(self):
                 f"is not found in the pool '{rbd_pool_name}'"
             )
 
-    def validate_subvolume_group(self):
```

Comment:
Will this script also create a default subvolumegroup with the name csi? if not, we need to consider creating it. The pinning is required when you have active mds; you might need to provide a command to disable/enable pinning

Reply:
>Will this script also create a default subvolumegroup with the name csi? 

Yaa it is calling the ` self.get_or_create_subvolume_group(subvolume_group, cephfs_filesystem_name)` so yes it will create it 

>The pinning is required when you have active mds; you might need to provide a command to disable/enable pinning
Does that mean we should check the `mds state` and then only run the pinning command?
I don't think if a pinning is enabled we can disable using the script as the script is run during creation and upgrading.